src/data/set/basic.lean

```diff
@@ -768,6 +768,10 @@ by finish [ext_iff]
 @[simp] lemma sep_univ {α} {p : α → Prop} : {a ∈ (univ : set α) | p a} = {a | p a} :=
 by { ext, simp }
 
+lemma subset_singleton_iff {α : Type*} {s : set α} {x : α} : s ⊆ {x} ↔ ∀ y ∈ s, y = x :=
```

Comment:
Should this be a `simp` lemma?

Reply:
BTW, `iff.rfl` should prove this lemma.

Reply:
I tagged the lemma with ```simp``` and proved it by ```iff.rfl```.

Thanks!